src/main/java/com/github/manolo8/darkbot/core/utils/EntityList.java

```diff
@@ -61,13 +62,13 @@ public class EntityList extends Updatable implements EntitiesAPI {
     public final List<Pet> pets                     = register(PET);
     public final List<BattleStation> battleStations = register(CBS_ASTEROID, CBS_MODULE, CBS_STATION, CBS_MODULE_CON, CBS_CONSTRUCTION);
     public final List<BasePoint> basePoints         = register(BASE_HANGAR, BASE_STATION, HEADQUARTER, QUEST_GIVER, BASE_TURRET, REPAIR_STATION, REFINERY);
-    public final List<Entity> unknown               = register();
+    public final List<Entity> unknown               = register(UNKNOWN);
```

Comment:
this change is pretty useless, no entity is ever created as UNKNOWN, as the predicate for it always returns false!!!

I'm actually not even sure why unknown is part of entity factory, as it can never actually occur (there's a fallback in EntityFactory#find which returns an in-line EntityBuilder `id, addr -> new unknown` but these will never actually be part of the UNKNOWN factory). Additionally there's extra logic in EntityFactory#sendEntity, where if it's not a factory (ie, it's not any of the ones in the list) it goes to the fallback listener, which is what does get stuff added here.

TLDR: should we just get rid of UNKNOWN entity factory type? i don't think it serves a purpose (the list already gets populated independently of that)